ecoscope/io/earthranger.py

```diff
@@ -708,6 +579,28 @@ def get_events(
         df.sort_values("time", inplace=True)
         return df
 
+    def get_patrols(self, filter=None, status=None, **addl_kwargs):
+        """
+        Parameters
+        ----------
+        filter:
```

Comment:
Can we expand the single filter parameter to multiple params in the signature? I find the use of a filter here confusing compared with our other function definitions